### PR TITLE
Issue 91

### LINKED ISSUE #322: [91] Issue #91 - Oct 12, 2017
To contribute to this issue, simply leave a comment here. Please also review [our contributing guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/CONTRIBUTING.md).

The current draft for this issue in [`_drafts/`](https://github.com/SwiftWeekly/swiftweekly.github.io/tree/master/_drafts). If you want to contribute directly, feel free to [open a pull request](https://github.com/SwiftWeekly/swiftweekly.github.io/compare?expand=1).

### PR DESCRIPTION
**Pull request by BasThomas**
Fixes #322

### CODE REVIEW

**Comment by jessesquires** [comment]
Looks great!

Need to add this one proposal:
https://lists.swift.org/pipermail/swift-evolution-announce/2017-September/000404.html

**Comment by jessesquires** [comment]
I can wrap up in a few and merge 😄

File: _posts/2017-10-12-issue-91.md [hunk 1]
```diff
@@ -42,7 +40,7 @@ Chris Lattner [opened a pull request](https://github.com/apple/swift/pull/12338)
 
 Recursive protocol constraints are now available on [the master toolchain](https://twitter.com/AirspeedSwift/status/918159453467295744)!
 
-### Proposals
+### Accepted proposals
```
Comment (BasThomas) [inline _posts/2017-10-12-issue-91.md]: Oh oops.

Comment (jessesquires) [reply _posts/2017-10-12-issue-91.md]: All good! 😄 

awesome issue! 🎉

File: _posts/2017-10-12-issue-91.md [hunk 2]
```diff
@@ -24,12 +24,10 @@ Kubra Mracek improved the display of fatal errors and wrote a blog post [on Swif
...
+Tony Allevato's [SE-0185: *Synthesizing `Equatable` and `Hashable` conformance*](https://github.com/apple/swift-evolution/blob/master/proposals/0185-synthesize-equatable-hashable.md) [was merged](https://github.com/apple/swift-evolution/blob/master/proposals/0185-synthesize-equatable-hashable.md)! This is huge — it provides automatic `Equatable` and `Hashable` conformance for types for whose members are all `Equatable` or `Hashable`. 😱🎉
 
 Greg Spiers [implemented](https://github.com/apple/swift/pull/11744) [SE-0186: *Remove ownership keyword support in protocols*](https://github.com/apple/swift-evolution/blob/master/proposals/0186-remove-ownership-keyword-support-in-protocols.md).
 
-Itai Ferber [implemented](https://github.com/apple/swift/pull/9005) [SE-0167: *Swift Encoders*](https://github.com/apple/swift/pull/9005).
```
Comment (BasThomas) [inline _posts/2017-10-12-issue-91.md]: Oh damn - got this from https://github.com/apple/swift-evolution/commit/121a41f101a1d2ba60cddf90567f68608a4957a8. Missed it was so old. Sorry!

Comment (jessesquires) [reply _posts/2017-10-12-issue-91.md]: no worries! 😊

**Pull request merged**